ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/XARecoveryModuleUnitTest.java

```diff
@@ -477,21 +481,18 @@ public int prepare(Xid xid) throws XAException {
 
                 @Override
                 public Xid[] recover(int i) throws XAException {
-                    count++;
-                    if (count == 1 || count == 5) {
-                        return new Xid[]{xid};
-                    } else if (count > 5) {
-                        return new Xid[0];
-                    } else {
-                        throw new XAException();
+                    recoverCount++;
+                    if (recoverCount == 1 || recoverCount == 3 || recoverCount == 5) {
+                        return new Xid[] { xid }; // TMSTARTRSCAN
+                    }else if (recoverCount > 5) {
```

Comment:
The code reformat has dropped the space between `}` and `else` which makes it inconsistent with the rest of the code in this file.

Reply:
Thanks, updated.